subsys/bluetooth/mesh/transport.h

```diff
@@ -77,6 +84,43 @@ struct bt_mesh_ctl_friend_sub_confirm {
 	uint8_t xact;
 } __packed;
 
+struct bt_mesh_ctl_path_request {
+	uint8_t  octer;
+	uint8_t  path_origin_forwarding_number;
+	uint8_t  path_origin_path_metric;
+	uint16_t dest;
+	uint16_t path_origin;
+	uint8_t  path_origin_addr_range;
+	uint16_t depend_origin;
+	uint8_t  dependent_origin_range;
+} __packed;
+
+struct bt_mesh_ctl_path_reply {
+	uint8_t  octer;
+	uint16_t path_origin;
+	uint8_t  path_origin_forwarding_number;
+	uint16_t path_target;
+	uint8_t  path_target_range;
+	uint16_t depend_target;
+	uint8_t  depend_target_range;
+} __packed;
+
+struct bt_mesh_ctl_path_comfirm {
+	uint16_t path_origin;
+	uint16_t path_target;
+} __packed;
+
+struct bt_mesh_ctl_depend_node_update {
+	uint8_t  octer;
+	uint16_t path_endpoint;
+	uint16_t dependent_addr;
+	uint8_t  dependent_range;
+} __packed;
+
+struct bt_mesh_ctl_path_req_solicitation {
+	uint16_t addr_list[5];
```

Comment:
I want to keep the status quo because there are already existing references.https://github.com/zephyrproject-rtos/zephyr/blob/main/subsys/bluetooth/mesh/transport.h#L73